src/mame/drivers/spec128.cpp

```diff
@@ -282,6 +282,10 @@ void spectrum_128_state::spectrum_128_mem(address_map &map)
 	map(0x4000, 0x7fff).rw(FUNC(spectrum_128_state::spectrum_128_ram_r<1>), FUNC(spectrum_128_state::spectrum_128_ram_w<1>));
 	map(0x8000, 0xbfff).rw(FUNC(spectrum_128_state::spectrum_128_ram_r<2>), FUNC(spectrum_128_state::spectrum_128_ram_w<2>));
 	map(0xc000, 0xffff).rw(FUNC(spectrum_128_state::spectrum_128_ram_r<3>), FUNC(spectrum_128_state::spectrum_128_ram_w<3>));
+
+	//FIXME must be available for children. review/replace templates usage
+	FUNC(spectrum_128_state::spectrum_128_ram_r<0>);
+	FUNC(spectrum_128_state::spectrum_128_ram_w<0>);
```

Comment:
You’re supposed to use [explicit template instantiations](https://docs.microsoft.com/en-us/cpp/cpp/explicit-instantiation?view=msvc-170) to explicitly instantiate templates, not hacks like this.